MSAL/src/requests/MSALSilentRequest.h

```diff
@@ -36,6 +36,7 @@
 
 - (id)initWithParameters:(MSALRequestParameters *)parameters
             forceRefresh:(BOOL)forceRefresh
+              tokenCache:(MSIDSharedTokenCache *)tokenCache
```

Comment:
nit: feels a bit awkward we need to pass token cache everywhere

Reply:
we need to pass it to requests only...